src/main/java/org/entando/entando/aps/system/services/widgettype/model/WidgetDto.java

```diff
@@ -46,7 +46,7 @@
     private boolean locked;
     private String widgetCategory;
 
-    protected class GuiFragmentRef {
```

Comment:
Is this change still required?

Reply:
Yes, I need to be able to instantiate this class during tests